src/components/home/institutionsCarousel/institutionsCarousel.tsx

```diff
@@ -1,55 +1,22 @@
 import { Col, Row } from "react-bootstrap";
-import { Autoplay, Navigation, Pagination } from "swiper";
-import "swiper/css";
-import "swiper/css/navigation";
-import "swiper/css/pagination";
-import { Swiper, SwiperSlide } from "swiper/react";
 import { importAllFilesFromFolder } from "../../../utils/utils";
 
 const InstitutionsCarousel = () => {
   const listInstitutionImages: any = importAllFilesFromFolder(
```

Comment:
rename file to reflect that the carousel has been removed?

Reply:
Done!